README.md

```
@@ -3,18 +3,22 @@
 > A fast, minimal low-level build system with intelligent target management
 
 [![License](https://img.shields.io/badge/license-MIT-blue.svg)](LICENSE)
-[![Version](https://img.shields.io/badge/version-0.1.5-green.svg)](autocc.cc)
+[![Version](https://img.shields.io/badge/version-0.1.6-green.svg)](autocc.cc)
 [![Build Status](https://img.shields.io/badge/build-passing-brightgreen.svg)]()
```

markdownlint-cli2 (0.17.2)

No empty links

(MD042, no-empty-links)